.github/workflows/bindings_ci.yml

```diff
@@ -109,3 +109,9 @@ jobs:
 
       - name: Build Framework
         run: target/debug/xtask swift build-framework --only-target=aarch64-apple-ios
+
+  complement-crypto:
+    name: "Run Complement Crypto tests"
+    uses: matrix-org/complement-crypto/.github/workflows/single_sdk_tests.yml@main
```

Comment:
I'm worried that we don't use a fixed/pinned version here, because that means that now our CI may fail because of external causes only (if complement-crypto fails because a new test was pushed to main and it failed, then our CI fails too). On the other hand, using a pinned version would mean bumping here. What are your thoughts on this?

Reply:
It's fully intended that as new tests are added to complement crypto that they are tested in rust SDK CI. Having to make a new PR to include these new tests adds friction which will decrease the likelihood of these tests being regularly run.

I share your concern though, which is why the complement crypto repo [also runs the same CI against rust-sdk main](https://github.com/matrix-org/complement-crypto/blob/main/.github/workflows/tests.yaml#L21) so if a new test fails in rust SDK, we'll know about it when the complement crypto PR is made, rather than waiting for things to break in rust SDK CI.

Reply:
Is it a problem to release Complement often, like very frequently?

Reply:
Sounds good, we can try it out like this and advise later if needs be. Thanks for the reasoning.